### PR TITLE
feat(config): add configuration for Smartly Wheel Controller

### PR DESCRIPTION
**Pull request by johnnyegel**
Added configuration description for Smartly AS - Wheel Controller, which was missing from the configuration collection for this vendor. Configuration reference and manual for this device can be found on the Z-Wave alliance web pages:
https://products.z-wavealliance.org/products/2853?selectedFrequencyId=1

### CODE REVIEW

File: packages/config/config/devices/0x024f/fwc01.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+			"unit": "/100s",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: We usually do this (consistency)
```suggestion
			"unit": "0.01s",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "11",
+			"label": "Dimmer mode",
+			"description": "Dimmer Operation Mode",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json:58]: What do these values mean? There should probably be options describing them.

File: packages/config/config/devices/0x024f/fwc01.json [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+			"label": "Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 15
+		},
+		{
+			"#": "14",
+			"label": "UI modes and Zones",
+			"description": "Enable UI Modes and Zones via Bits",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: This sounds like the parameter should be converted to [partials](https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters) (one per bit)

File: packages/config/config/devices/0x024f/fwc01.json [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+			"minValue": 0,
+			"maxValue": 67,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "Light Zone Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 30,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: What do these values mean? There should probably be options describing them.

File: packages/config/config/devices/0x024f/fwc01.json [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Back Unit Role",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json:264]: What do these values mean? There should probably be options describing them.

File: packages/config/config/devices/0x024f/fwc01.json [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Back Unit Light Zone",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 4,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json:286]: What do these values mean? There should probably be options describing them.

File: packages/config/config/devices/0x024f/fwc01.json [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+			"defaultValue": 1
+		},
+		{
+			"#": "32",
+			"label": "Thermostat Set Point Max",
+			"description": "Set Point Max in 1/10th of a degree Celcius",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+			"unit": "1/10 degrees",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 11]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 270
+		},
+		{
+			"#": "34",
+			"label": "Thermostat Set Point Min",
+			"description": "Set Point Min in 1/10th of a degree Celcius",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 12]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 50
+		},
+		{
+			"#": "36",
+			"label": "Thermostat Control Resend Interval",
+			"description": "Resend Interval in seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 13]
```diff
@@ -0,0 +1,227 @@
...
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 3600
+		},
+		{
+			"#": "37",
+			"label": "Thermostat Minimum On/Off Interval",
+			"description": "Minimum On/Off interval in seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 14]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "38",
+			"label": "Thermostat Floor Temp Max",
+			"description": "Floor Temp Max in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 15]
```diff
@@ -0,0 +1,227 @@
...
+			"label": "Cold Start Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 20
+		},
+		{
+			"#": "64",
+			"label": "Temperature Report Inteval",
+			"description": "Report Interval in seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]: unnecessary description

File: packages/config/config/devices/0x024f/fwc01.json [hunk 16]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "66",
+			"label": "Temperature Report Threshold",
+			"description": "Report Threshold in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 17]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 10
+		},
+		{
+			"#": "67",
+			"label": "Thermostat On∕Off Interval Override Threshold",
+			"description": "Temperature Threshold in 1/10th of a degree Celcius",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 18]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 0
+		},
+		{
+			"#": "240",
+			"label": "Floor Sensor Temperature Offset",
+			"description": "Temperature Offset in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 19]
```diff
@@ -0,0 +1,227 @@
...
+			"valueSize": 1,
+			"minValue": -128,
+			"maxValue": 127,
+			"defaultValue": 0
+		},
+		{
+			"#": "241",
+			"label": "Room Sensor Temperature Offset",
+			"description": "Temperature Offset in 1/10th of a degree Celcius",
+			"unit": "1/10 degrees",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x024f/fwc01.json [hunk 20]
```diff
@@ -0,0 +1,227 @@
...
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 270
+		},
+		{
+			"#": "40",
+			"label": "Temperature Sensor Select",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/fwc01.json:357]: What do these values mean? There should probably be options describing them.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x024f/fwc01.json (hunk 1 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: I agree. I'll remove it.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 2 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Ok, I did not know this, so I'll fix it. I used an existing config template for a similar device from the same vendor as the base for this one. This was from there.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 3 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Yes, agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 4 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json:58]: Yes, I'll correct that. Once more I used another config as a template, which had defined this config like this (see fms01.json in the same folder as this).

File: packages/config/config/devices/0x024f/fwc01.json (hunk 5 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Yes, that looks like a very good way to define these options. I will fix it.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 6 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Yes, I'll fix that.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 7 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json:264]: I'll fix those as well.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 8 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json:286]: I'll correct it.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 11 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 12 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 13 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 14 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 20 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json:357]: I'll add options.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 15 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 16 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 17 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 18 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

File: packages/config/config/devices/0x024f/fwc01.json (hunk 19 above)
Comment (johnnyegel) [reply packages/config/config/devices/0x024f/fwc01.json]: Agreed.

**Comment by johnnyegel** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]
Approved with my changes

**Pull request merged**